.github/ISSUE_TEMPLATE/4_KIT_release_checks.md

```diff
@@ -0,0 +1,43 @@
+---
+name: KIT Release Checks
+about: Use this template to track KIT release checks
+title: "[KIT NAME] Release Checks"
+labels: kit
+assignees: kelaja
```

Comment:
Same comment regarding specific GH user in template as for the release checks template.

Reply:
[70def77](https://github.com/eclipse-tractusx/sig-release/pull/479/commits/70def77137e81d7d94dd1d6b60b624f0ee98c6ef)